AGENTS-REACT.md

```
@@ -0,0 +1,671 @@
+pnpm preview          # Preview build
+
+# Testing (manual)
+# Open http://localhost:4321/game
+# Test platforming, combat, visuals
+```
+
+---
+
+## File Organization
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)